### ISSUE TITLE
Clarification Required

### ISSUE THREAD

**Issue by Air-Git** [MicrosoftDocs/memdocs#1984]
I know this is an old topic, but I think the article is still unclear. There are several distinct points:
1) Does a policy node apply to Device, User, or both? e.g. Policy CSP Device Lock applies to devices, Policy CSP Experience applies to users.
2) What happens if a _user_ is assigned a _device_ setting, and vice versa? My understanding is that "User" actually means "Primary User of the device, assigned at enrollment". So a device policy assigned to a user will be applied to the device of which the user is the Primary user.
3) Are there any policies that do not work if cross-assigned i.e. user policy assigned to device, or device policy assigned to user?
4) If there is a setting applied to the device, and to the user, which takes precedence?

In perhaps 90% of cases, we will want a setting to apply to All Devices or All Users, so it won't make a difference, as long as it works. There are some obvious corner cases like kiosks, but that's not difficult to understand. But it is much easier to have an exclusion for a user group than a device group. So my preference would be normally to assign policies to users except for very specific hardware dependent policies like BitLocker or Firmware.

I think the examples you have given are the corner cases, where the approach is fairly obvious. What is needs to clarity on what happens when you assign a device setting to users, or a user setting to devices.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cb9f0c71-ac8b-b047-9934-1bffb895e75c
* Version Independent ID: 44f25bbe-ccc6-5480-09b4-cfa18832b0bb
* Content: [Assign device profiles in Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/configuration/device-profile-assign)
* Content Source: [memdocs/intune/configuration/device-profile-assign.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/configuration/device-profile-assign.md)
* Service: **microsoft-intune**
* Sub-service: **configuration**
* Technology: ****
* GitHub Login: @MandiOhlinger
* Microsoft Alias: **mandia**

**Comment by MandiOhlinger** [comment]
@Air-Git Good questions! If you're still looking for some guidance, we suggest looking at [Settings catalog: Device scope vs. user scope settings](https://docs.microsoft.com/mem/intune/configuration/settings-catalog#device-scope-vs-user-scope-settings). We also added some info based on your last question.

We also updated [Assign user and device configuration profiles in Microsoft Intune](https://docs.microsoft.com/mem/intune/configuration/device-profile-assign) so users can find this info easier.

The doc changes are live.
